src/client/container/transfercall/contactfield.js

```diff
@@ -2,24 +2,51 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import styles from './transfercall.css';
+import { truncate } from '../../utils/acutils';
+
+class EachContact extends React.Component {
```

Comment:
EachContact component can be export to be a shared component to be used in both(or more) places